platform/feature-api/communication/src/remote-pod/remote.ts

```diff
@@ -106,10 +106,11 @@ export const podBubblewrapClasses: Classes = {
     "@polypoly-eu/rdf.Quad": polyQuad,
 };
 
+export const podBubblewrap = Bubblewrap.create(podBubblewrapClasses);
```

Comment:
I know you're removing duplication, but this makes me a bit uneasy - I don't know in which situations it may be important for this instance to be created lazily (how it was), as opposed to upfront (how it would be now). Couldn't you just import `Bubblewrap.create` in the tests and call it there? That way you wouldn't have to change any semantics.

Reply:
There's no API to change the classes that can be wrapped in runtime, and there probably shouldn't be. Think about it, creating a type on the fly and making it roundtrippable through the remote-pod. Probably not a good idea.
From another point of view: can you create *classes* lazily? Probably not...

Reply:
I'm not sure I follow entirely - what this line does is create an _instance_ of the `Bubblewrap` class, with a list of known types to use for serialisation.

That said, I took a closer look at the `Bubblewrap` class and the only relevant state it has seems to be the list of classes, and the _codec_, which is only created once during initialisation. Now, I don't know what relevant state this _codec_ has, that would take a bit more time to fully review and understand this code.

My point is - why introduce a potential issue in code none of us really understands to save one line of code in the tests? It's a whitebox test to begin with, whether it checks `podBubblewrap` or `podBubblewrapClasses` seems inconsequential - in both cases it's verifying some internal state.

Reply:
Well, I'd say that we should have tests to detect issues if they arise. But it's true that this might not necessarily be part of the scope of this PR, which is notably about adding tests, not introducing new issues. So no problem in eliminating it. But I might want to create another PR, just for the sake of understanding the code.